pkgs/games/gdlauncher/default.nix

```
@@ -0,0 +1,30 @@
+{ pkgs, lib, ... }:
```

```suggestion
{ lib, fetchFromGitHub }:
```
this of course only works once you implemented the github wrapper currently it would have to be fetchurl. If you do it like that you don't need the `pkgs.` infront of it. The three dots aren't needed because the function only get's parameters it actually needs.